rs/minter/minter.did

```diff
@@ -55,6 +63,19 @@ type ReturnError = variant {
   Expired;
 };
 service : () -> {
+  accept_ownership : () -> ();
+  get_config_user_page_end : () -> (nat64) query;
+  get_version : () -> (CanisterLifecycle) query;
+  get_version_text : () -> (text) query;
+  grant_admin : (principal) -> ();
+  is_admin : (principal) -> (bool) query;
+  is_owner : (principal) -> (bool) query;
+  owner : () -> (opt principal) query;
+  pending_owner : () -> (opt principal) query;
+  renounce_admin : () -> ();
+  revoke_admin : (principal) -> ();
+  transfer_ownership : (opt principal) -> ();
+  transfer_ownership_immediate : (opt principal) -> ();
```

Comment:
How did you generate these declarations? The Makefile uses candid-extractor, but it doesn't seem to produce these functions.

Reply:
Those endpoints are exposed by the Rustic library. https://github.com/internet-computer-protocol/rustic#export-candid

Reply:
For example, `transfer_ownership` is defined here https://github.com/internet-computer-protocol/rustic/blob/653c4057687143db4d8d527f5e127e1fcd6e2b1e/src/access_control.rs#L124

Reply:
Interesting.. so it has to be combined manually? Hope this can be improved in the future ...